test/app/lib/crypto/helpers.ts

```diff
@@ -0,0 +1,20 @@
+import {ExtendedKey} from "../../../../app/lib/crypto/key/key"
+import {fromEntropy} from "../../../../app/lib/crypto/seed"
+import * as PrivateKey from "../../../../app/lib/crypto/key/privateKey"
+
+/**
+ * Helper for testing
+ * @param {string} seedEntropy
+ * @param {string} passPhrase
+ * @returns {ExtendedKey<PrivateKey>}
+ */
+export function getMasterKey(seedEntropy: string, passPhrase = "Bitcoin seed"):
```

Comment:
`passPhrase` should be `Witnet seed` by default. Only the tests use `Bitcoin seed` in order to recycle existing test vectors.